### PR TITLE
add seronet_dock

### PR DESCRIPTION
**Pull request by HannesBachter**

### CODE REVIEW

**Review by fmessmer — CHANGES_REQUESTED** [review]
please add some CI (`.travis.yml`)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+env:
+  global:
+    - ROS_REPO=ros
+    - NOT_TEST_INSTALL=true
+    - VERBOSE_TESTS=false
+    - UPSTREAM_WORKSPACE=file
+  matrix:
+    - ROS_DISTRO=kinetic
+install:
+  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
```
Comment (fmessmer) [inline .travis.yml]: this is not the latest default `.travis.yml` we use - where did you get that from?

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (fmessmer) [reply .travis.yml]: use https://github.com/mojin-robotics/mojin_drivers/blob/kinetic-devel/.travis.yml as an example

Comment (HannesBachter) [reply .travis.yml]: I gave it another try

**Review by fmessmer — APPROVED** [review]
you're welcome

**Pull request merged**